tiling.js

```diff
@@ -1872,6 +1872,11 @@ export const Spaces = class Spaces extends Map {
 
         // Initialize spaces _after_ monitors are set up
         this.forEach(space => space.init());
+
+        // Bind to visible workspace when starting up
+        signals.disconnect(Main.panel);
```

Comment:
We don't need to disconnect signals here.  `init` method is called on PaperWM startup - and all signals are destroyed when PaperWM is disabled (including during lockscreen).